nan.h

```diff
@@ -1402,7 +1402,7 @@ class NanCallback {
     return NanNew(handle)->Get(kCallbackIndex)->IsUndefined();
   }
 
-  v8::Handle<v8::Value> Call(int argc, v8::Handle<v8::Value> argv[]) const {
+  v8::Handle<v8::Value> Call(int argc, v8::Handle<v8::Value> argv[], v8::Handle<v8::Object> target=NanGetCurrentContext()->Global()) const {
```

Comment:
Yes, I too thought that the arguments might be better of in the other order.

Since 0.10, `node::MakeCallback` does take `v8::Value` if it can be cast to `v8::Object`. Do you mean v8 now accepts any v8::Value? If it was not possible in the past, we probably cannot implement it anyway.